### PR TITLE
Fix `UI_ArenaBattleLoadingScreen` prefab

### PR DESCRIPTION
**Pull request by boscohyun**
### Related Links

1. [아레나 전투 로딩 화면 이미지 깨짐](https://app.asana.com/0/1141562434100787/1200887083113949/f)
2. [아레나 보드에서 상단메뉴 뜨고, 전투에서는 상단메뉴 안떠야 하는데 반대로 동작함](https://app.asana.com/0/1141562434100787/1200888226950609/f)

### Screenshot

image
image
image

### CODE REVIEW

**Review by Namyujeong — CHANGES_REQUESTED** [review]
Only this screen is missing the backspace keyboard mapping. (It seems to have flown while reviving the menu.)

스크린샷 2021-09-06 오후 8 04 12

**Comment by boscohyun** [comment]
> Only this screen is missing the backspace keyboard mapping. (It seems to have flown while reviving the menu.)

@Namyujeong Fixed it.

**Review by Namyujeong — APPROVED** [review]
스크린샷 2021-09-07 오후 8 05 30
스크린샷 2021-09-07 오후 8 06 02

File: nekoyume/Assets/_Scripts/UI/HeaderMenu.cs [hunk 1]
```diff
@@ -144,6 +144,25 @@ public override void Initialize()
...
+                    CloseWidget = () => { menuToggleDropdown.isOn = false; };
+                    WidgetStack.Push(gameObject);
+                }
+                else
+                {
+                    CloseWidget = null;
+                    Observable.NextFrame().Subscribe(_ =>
+                    {
+                        var list = WidgetStack.ToList();
+                        list.Remove(gameObject);
```
Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/HeaderMenu.cs:158]: Is this widget clearly removed in `WidgetStack` this line?

Comment (boscohyun) [reply nekoyume/Assets/_Scripts/UI/HeaderMenu.cs:158]: @Unengine Yes. I had test that. Is there anything might be a problem?

**Review by hskim881028 — APPROVED** [review]

File: nekoyume/Assets/_Scripts/UI/HeaderMenu.cs (hunk 1 above)
Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/HeaderMenu.cs:158]: I just worried about converting stack to list and modifying the value causes no problem. If it is working right, i think it is ok.

**Review by Unengine — APPROVED** [review]

**Pull request merged**